graylog2-server/src/test/java/org/graylog/events/contentpack/facade/EventDefinitionFacadTest.java

```diff
@@ -0,0 +1,329 @@
+        final AggregationConditions condition = AggregationConditions.builder()
+                .expression(Expr.And.create(trueExpr, trueExpr))
+                .build();
+        final AggregationEventProcessorConfigEntity aggregationConfig = AggregationEventProcessorConfigEntity.builder()
+                .query(ValueReference.of("author: \"Jane Hopper\""))
+                .streams(ImmutableSet.of())
+                .groupBy(ImmutableList.of("project"))
+                .series(ImmutableList.of(serie))
+                .conditions(condition)
+                .executeEveryMs(ValueReference.of(122200000L))
+                .searchWithinMs(ValueReference.of(1231312123L))
+                .build();
+
+        final EventDefinitionEntity eventDefinitionEntity = EventDefinitionEntity.builder()
+                .title(ValueReference.of("title"))
+                .description(ValueReference.of("description"))
+                .priority(ValueReference.of(1))
+                .config(aggregationConfig)
+                .alert(ValueReference.of(true))
+                .fieldSpec(ImmutableMap.of("fieldSpec", fieldSpec))
+                .keySpec(ImmutableList.of("keyspec"))
+                .notificationSettings(EventNotificationSettings.builder()
+                        .gracePeriodMs(123123)
+                        .backlogSize(123)
+                        .build())
+                .notifications(ImmutableList.of(EventNotificationHandlerConfigEntity.builder()
+                        .notificationId(ValueReference.of("123123"))
+                        .build()))
+                .storage(ImmutableList.of())
+                .build();
+
+        final JsonNode data = objectMapper.convertValue(eventDefinitionEntity, JsonNode.class);
+        return EntityV1.builder()
+                .data(data)
+                .id(ModelId.of("beef-1337"))
+                .type(ModelTypes.EVENT_DEFINITION_V1)
+                .build();
+    }
+
+    @Test
+    public void createNativeEntity() {
+        final EntityV1 entityV1 = createTestEntity();
+        final NotificationDto notificationDto = NotificationDto.builder()
+                .config(HTTPEventNotificationConfig.builder().url("https://hulud.net").build())
+                .title("Notify me Senpai")
+                .description("A notification for senpai")
+                .id("dead-beef")
+                .build();
+        final EntityDescriptor entityDescriptor = EntityDescriptor.create("123123", ModelTypes.NOTIFICATION_V1);
+        final ImmutableMap<EntityDescriptor, Object> nativeEntities = ImmutableMap.of(
+                entityDescriptor, notificationDto);
+
+        final JobDefinitionDto jobDefinitionDto = mock(JobDefinitionDto.class);
+        final JobTriggerDto jobTriggerDto = mock(JobTriggerDto.class);
+        when(jobDefinitionDto.id()).thenReturn("job-123123");
+        when(jobSchedulerClock.nowUTC()).thenReturn(DateTime.now());
```

Comment:
The `DateTime.now()` call requires a timezone. (Jenkins build complained)

```suggestion
        when(jobSchedulerClock.nowUTC()).thenReturn(DateTime.now(DateTimeZone.UTC));
```